external-crates/move/crates/move-bytecode-source-map/src/utils.rs

```diff
@@ -5,12 +5,19 @@
 use crate::source_map::SourceMap;
 use anyhow::{format_err, Result};
 use move_ir_types::location::Loc;
-use std::{fs::File, io::Read, path::Path};
+use std::{
+    fs::File,
+    io::{Read, Write},
+    path::Path,
+};
 
 pub type Error = (Loc, String);
 pub type Errors = Vec<Error>;
 
 pub fn source_map_from_file(file_path: &Path) -> Result<SourceMap> {
+    if file_path.extension().is_some_and(|ext| ext == "json") {
```

Comment:
nit ,constant maybe

Reply:
constants are used elsewhere for extension though I understand that `json` is not like the others.
But it is a fair comment

Reply:
My only request is that if we extract this into a constant, that it's the `SOURCE_MAP_EXT`, and not `JSON_EXT`.